src/EnergyPlus/DXCoils.cc

```diff
@@ -12525,8 +12519,8 @@ void CalcMultiSpeedDXCoilCooling(EnergyPlusData &state,
 
     // Using/Aliasing
     using Curve::CurveValue;
-    auto &MSHPMassFlowRateHigh = state.dataHVACGlobal->MSHPMassFlowRateHigh;
-    auto &MSHPMassFlowRateLow = state.dataHVACGlobal->MSHPMassFlowRateLow;
+    Real64 MSHPMassFlowRateHigh = state.dataHVACGlobal->MSHPMassFlowRateHigh;
```

Comment:
More opportunistic fixing of bad `auto &`, if they are not modified they should be value shortcuts.